content/release-notes/2023-01-10.md

```diff
@@ -0,0 +1,8 @@
+---
+label: 'Storage'
+---
+
+### What's new
+
+- Pageserver is now capable of downloading layer files on demand.
+  This feature is the foundation for offloading cold data to S3, enabling a cost-efficient offering with bottomless storage.
```

Comment:
on demand download allows fast redistribution of tenants across pageservers fleet and recovery from local state loss.
It will augment offloading cold data by allowing resurrection of it on demand